php/commands/search-replace.php

```diff
@@ -124,6 +128,12 @@ public function __invoke( $args, $assoc_args ) {
 					WP_CLI::error( sprintf( 'Unable to open "%s" for writing.', $assoc_args['export'] ) );
 				}
 			}
+			$export_insert_size = \WP_CLI\Utils\get_flag_value( $assoc_args, 'export_insert_size' );
```

Comment:
Once you specify the default value per my comment above, you can just access `$assoc_args['export_insert_size']` directly.